airflow/config_templates/config.yml

```diff
@@ -2317,7 +2317,7 @@ scheduler:
       default: "10.0"
     catchup_by_default:
       description: |
-        Turn off scheduler catchup by setting this to ``False``.
+        Turn on scheduler catchup by setting this to ``True``.
         Default behavior is unchanged and
         Command Line Backfills still work, but the scheduler
```

Comment:
Not exactly related to this change, but what does *Default behavior is unchanged* even mean here…

Reply:
I... am not sure 😅 I was trying to keep the changes to a minimum but happy to remove that line.